build-system/npm-publish/write-package-files.js

```diff
@@ -186,11 +185,10 @@ async function writePackageJson({useBentoCore}) {
       preact: '^10.2.1',
       react: '^17.0.0',
     },
+    dependencies: {
+      [corePkgName]: corePkgVersion,
```

Comment:
remove `[ ]`

Reply:
But, we need it? `corePkgName` is a variable and this lets us set a map that says `dependencies {'@bentoproject/core': 0.x}`

Reply:
oh oops!